### ISSUE TITLE
about cldice loss

### ISSUE THREAD

**Issue by long123524** [dmitrysarov/clDice#3]
How to train it as a loss function? Isn't the loss function generally required to be defined as a class? , If I use the code you provided for training, the following error will occur. How can I correct this error?
![image](https://user-images.githubusercontent.com/73996375/128622033-04ae0bba-f856-4eb9-8bfb-79767a254468.png)
The calculated loss value of cldice is not a scalar, but a tensor with a shape of (4, 2), so the above error occurred. How should I correct this error?

**Comment by dmitrysarov** [comment]
Hi @long123524. Could you send me the whole code to check?

**Comment by long123524** [comment]
The complete code has been sent to your e-mail, can you tell me the solution to this problem?

**Comment by dmitrysarov** [comment]
I haven't  received anything

**Comment by long123524** [comment]
Could you please provide the email address, I’m sorry, it should have been sent wrong before

**Comment by dmitrysarov** [comment]
just drop it here, mb it will be useful for others
